app/jinja_filters.py

```diff
@@ -98,3 +132,8 @@ def format_number_to_alphabetic_letter(number):
 @blueprint.app_context_processor
 def utility_processor():
```

Comment:
Might be worth updating this name to make it more obvious while we are making changes here.